### ISSUE TITLE
Add charging stations for electric cars and e-bike charges on the map

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#1437]

**Comment by vng** [comment]
Now we show this tag https://taginfo.openstreetmap.org/tags/amenity=charging_station on the map. But without distinguishing on bike/car.

**Comment by pastk** [comment]
Our current icon:

![Screenshot_1643641662](https://user-images.githubusercontent.com/18434508/151818313-e84af6e8-066f-4b08-8c2a-f1cfe4aa73de.png)

Shall we include charging stations into the `@fuel` search category? @biodranik

**Comment by toas-koas** [comment]
Is there any update?

**Comment by biodranik** [comment]
@pastk no, let's split charging stations. Gas car owners don't expect to see charging stations, and e-cars drivers don't expect to see petrol stations.
It is critically important to distinguish between car and bicycle chargers. @TheAdventurer64 can you help?

**Comment by TheAdventurer64** [comment]
> @pastk no, let's split charging stations. Gas car owners don't expect to see charging stations, and e-cars drivers don't expect to see petrol stations.
> 
> It is critically important to distinguish between car and bicycle chargers. @TheAdventurer64 can you help?

I checked into how often `amenity=charging_station` + the`bicycle` tag were used, and it looks like they were only combined 12 times. In addition, neither Google or Apple Maps supports special presets for e-bike charging stations. IMHO, we should probably wait until the feature gets more popular before we support it on Organic Maps.

**Comment by biodranik** [comment]
There are [4.5K](https://taginfo.openstreetmap.org/tags/amenity=charging_station#combinations) bicycle=yes and 1K motorcycle=yes charging stations in taginfo. How did you check?

We can split car, bike, and motorbike chargers into different types, so users can see their combinations when selecting one object (we have priority icon support, right? e.g. draw a car charger icon if there's also bicycle and/or motorcycle charging there). However, I see an issue with the editor on how to add these types. Creating two separate points is not a good idea. @vng any ideas?

**Comment by vng** [comment]
Have only 2 types in hierarchy: amenity-charging_station and amenity-charging_station-motorcar, because for the _fast_ display (select styles). All other stuff like bicycle, socket, operator, opening_hours goes into metadata.

For the fee and access I have a separate refactoring in mind, that will affect all POI types:
https://github.com/organicmaps/organicmaps/issues/1312

**Comment by tiptoptom** [comment]
Right now there's a charging station shown as bicycle charging station although both is tagged, motorcar=yes and bicycle=yes

**Comment by pastk** [comment]
We have 3 feature types already:
- "Charging Station"
- "Bicycle Charging Station" (bicycle=yes)
- "Car Charging Station" (motorcar=yes)

If a station has both bicycle=yes & motorcar=yes then it'll have 2 types.

All use the same icon at the moment.
